test/e2e/handler/utils.go

```diff
@@ -298,6 +298,17 @@ func interfacesForNode(node string) AsyncAssertion {
 	}, ReadTimeout, ReadInterval)
 }
 
+func waitForFreshNodeNetworkState(node string) {
+	// Worst case is 5 seconds so let's just use time.now()
```

Comment:
You are commenting on an issue that is not visible in the code (at least not at this line). Could you please elaborate what is the issue in more detail? I don't see an issue using `Now`